quinn-udp/src/unix.rs

```diff
@@ -257,6 +265,17 @@ impl UdpSocketState {
         self.may_fragment
     }
 
+    /// Sets the socket to receive packet receipt timestamps.
+    #[cfg(target_os = "linux")]
+    pub fn set_enable_rx_timestamps(&self, sock: UdpSockRef<'_>, enabled: bool) -> io::Result<()> {
```

Comment:
I think `_enable_` is misleading, given that I can call `set_enable_rx_timestamps(some_sock, false)`.

How about:

```suggestion
    pub fn set_rx_timestamps(&self, sock: UdpSockRef<'_>, enabled: bool) -> io::Result<()> {
```

Consistent with e.g. `set_gro` on Windows.